app/va/va_onsite/va_onsite_client.py

```diff
@@ -33,8 +33,8 @@ def post_onsite_notification(self, data: dict):
                                      data=json.dumps(data),
                                      headers=self._build_header())
 
-            self.logger.info(f'onsite_notifications POST response: status_code={response.status_code}, '
-                             f'json={response.json()}')
+            self.logger.info("onsite_notifications POST response: status_code=%i, json=%s",
```

Comment:
I think everything should be %s, even if we know it's an integer. I tested giving it an int (locally) and it automatically cast it to a string. I'd double check the documentation.

Reply:
Updated